pkg/controller/pcidevice/pcidevice_controller.go

```diff
@@ -72,31 +73,16 @@ func (h Handler) reconcilePCIDevices(hostname string) error {
 		if err != nil {
```

Comment:
when `err != nil`, it means `devCR` is nil,  can NOT `devCopy := devCR.DeepCopy()`